Doc/library/typing.rst

```diff
@@ -1985,6 +1985,15 @@ Functions and decorators
 
    .. versionadded:: 3.8
 
+   .. versionchanged:: 3.11
+      The decorator will now set the ``__final__`` attribute to ``True``
+      on the decorated object. Thus, a check like
+      ``if getattr(obj, "__final__", False)`` can be used at runtime
+      to determine whether an object ``obj`` has been marked as final.
+      If the decorated object does not support setting attributes,
+      the decorator returns the object unchanged.
```

Comment:
minor suggestion to improve clarity:
```suggestion
      the decorator returns the object unchanged without raising an exception.
```

Reply:
Thanks for the review, I applied both changes.